scst-scan/create-supply-chain-workload.md

```diff
@@ -0,0 +1,120 @@
+grype-app-scanning-catalog    │ │ └─TaskRun/resumption-task-499fk                                             -                          2m18s
+grype-app-scanning-catalog    │ │   ├─Pod/resumption-task-499fk-pod                                           False  PodCompleted        2m18s
+grype-app-scanning-catalog    │ │   └─ResolutionRequest/cluster-9f31100f32fba78553ef691245146126              -                          2m18s
+grype-app-scanning-catalog    │ └─TaskRun/resumption-task-499fk                                               -                          2m18s
+grype-app-scanning-catalog    │   ├─Pod/resumption-task-499fk-pod                                             False  PodCompleted        2m18s
+grype-app-scanning-catalog    │   └─ResolutionRequest/cluster-9f31100f32fba78553ef691245146126                -                          2m18s
+grype-app-scanning-catalog    └─Stage/stage-2-grype-image-scan-wslkz                                          -                          64s
+grype-app-scanning-catalog      └─PipelineRun/stage-2-grype-image-scan-7789q                                  -                          64s
+grype-app-scanning-catalog        ├─CustomRun/stage-2-grype-image-scan-7789q-scan                             -                          58s
+grype-app-scanning-catalog        │ └─ManagedResource/stage-2-grype-image-scan-7789q-scan-mr-xqc84            -                          58s
+grype-app-scanning-catalog        │   └─ImageVulnerabilityScan/golang-app-robin-test-bbrpz                    -                          58s
+grype-app-scanning-catalog        │     └─PipelineRun/golang-app-robin-test-bbrpz-sqsrp                       -                          58s
+grype-app-scanning-catalog        │       ├─PersistentVolumeClaim/pvc-160c7d4c76                              -                          58s
+grype-app-scanning-catalog        │       ├─TaskRun/golang-app-robin-test-bbrpz-sqsrp-scan-task               -                          39s
+grype-app-scanning-catalog        │       │ └─Pod/golang-app-robin-test-bbrpz-sqsrp-scan-task-pod             False  ContainersNotReady  39s
+grype-app-scanning-catalog        │       └─TaskRun/golang-app-robin-test-bbrpz-sqsrp-workspace-setup-task    -                          58s
+grype-app-scanning-catalog        │         └─Pod/golang-app-robin-test-bbrpz-sqsrp-workspace-setup-task-pod  False  PodCompleted        58s
+grype-app-scanning-catalog        ├─ResolutionRequest/cluster-ed022311336c4f218d4065723b853bfc                -                          64s
+grype-app-scanning-catalog        └─TaskRun/stage-2-grype-image-scan-7789q-prepare                            -                          64s
+grype-app-scanning-catalog          └─Pod/stage-2-grype-image-scan-7789q-prepare-pod                          False  PodCompleted        64s
+```
+
+### <a id="verify-workload-scanning"></a>Verify workload performed scanning by checking scan results
```

Comment:
**Question**: should this ID be `verify-workload-performed-scanning-by-checking-scan-results`?